### ISSUE TITLE
Technical

### ISSUE THREAD

**Issue by jsjang97** [Nina-Sitarski12/Project#7]
``
The beaver image is not loading! I think you should check if the source is correctly located.

**Comment by Nina-Sitarski12** [comment]
Yeah, I noticed this as well. It's strange because if I view in browser from visual studio, then the images show but if I follow the domain link then there are issues. I will look into this. Thanks!
